main/job-stats.js

```diff
@@ -1,7 +1,23 @@
 'use strict'
 
+const { InfluxDB, Point } = require('@influxdata/influxdb-client')
 const Store = require('electron-store')
 
+const client = new InfluxDB({
+  url: 'https://eu-central-1-1.aws.cloud2.influxdata.com',
+  // station-anonymous-write
+  token: 'P9INe7-DkMB_oYSAXgw0IlBcqRc4bDi05jZFz0NZd8aApv5lieqBiN59M8cmHs6yNMRtly8_KD9RYxnyrnx8dA=='
+})
+const writeClient = client.getWriteApi(
+  'julian.gruber@protocol.ai',
+  'station',
+  'ns'
+)
+
+setInterval(() => {
+  writeClient.flush()
+}, 1000)
+
```

Comment:
Let's move the integration with InfluxDB into a standalone file so that we can easily report telemetry from other places too. How about `main/telemetry-client.js` exporting `writeClient`?